README.md

```diff
@@ -1,5 +1,20 @@
 # Pyocient Release Notes
 
+## 3.0.2
+
+- The `TypeCodes` class has been changed to a python `IntEnum`
+- GIS types (STPoint, STLinestring, STPolygon) have been renamed from _STPoint, _STLinestring, _STPolygin
+- The `user` and `password` fields have been removed from the `Connection` object and are no longer stored in the object
+- A `security_token` from another connection can be passed in to the `connect` function
+- The `Session` object has been removed
+- The `custom_type_to_json` function has been moved into the pyocient API to allow consistent conversion of resultset data to JSON format
+- The `patch` portion of the pyocient version is now sent to the database and will be logged there
+- The `--nooutput` command line parameter has been fixed to not throw a python exception
+- An error reporting `local variable 'return_code' referenced before assignment` has been fixed
+- A `--rows` argument to the command line program has been added to allow loading resultset data in batches
+- An alpha version of OpenAI "text to SQL" has been added to the command line
```

Comment:
Should a new feature like this warrant a minor version bump - 3.1.0?

Reply:
yes

Reply:
What happened to `3.0.0`...? Why are we skipping versions?

Reply:
because sometimes versions get built and posted here: https://artifactory.corp.ocient.com/artifactory/tools_pypi/user_dboutcher_pyocient_version/pyocient/ that don't get published publicly

Reply:
It's **very** strange to me to bump the **public** version for internal-only releases (aka pre-releases). see: https://semver.org/spec/v2.0.0.html#spec-item-9

Reply:
perhaps when the tools team rewrites the python packaging to be more Stoyan friendly, they can additionally make it support non-numeric package versions